modules/instrument_manager/jsx/instrumentManagerIndex.js

```diff
@@ -145,8 +145,8 @@ class InstrumentManagerIndex extends Component {
             You do not have access to this page.
           </div>
         );
-      } else if (this.state.data.writable) {
-        let url = loris.BaseURL.concat('/instrument_manager/?format=json');
+      } else if (this.state.data.fieldOptions.writable) {
+        let url = loris.BaseURL.concat('/instrument_manager/Instrument');
```

Comment:
Did you forget to commit a file? There doesn't seem to be an instrument_manager/Instrument endpoint defined

Reply:
yes, I forget commit a file.